api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1035,6 +1035,7 @@ private HttpResponse execute(Method method, String region, String bucketName, St
 
     if (headerMap != null) {
       headerMap = normalizeHeaders(headerMap);
+
```

Comment:
We can remove the extra empty lines in middle

Reply:
done